project/Vagrantfile

```diff
@@ -31,16 +31,20 @@ end
 @properties = load_properties("project.properties")
 @ml_version = ENV['ML_VERSION'] || @properties['ML_VERSION'] || "7"
 @nr_hosts = (ENV['NR_HOSTS'] || @properties['NR_HOSTS'] || "3").to_i
+@memory = (ENV['VM_MEMORY'] || @properties['VM_MEMORY'] || "6144").to_i
+@cpus = (ENV['VM_CPUS'] || @properties['VM_CPUS'] || "4").to_i
 puts "ml-version=#{@ml_version}"
 puts "nr-hosts=#{@nr_hosts}"
+puts "vm-memory=#{@memory}"
+puts "vm-cpus=#{@cpus}"
 
-def get_vm_name(i)
-  File.basename(Dir.getwd) + "-ml" + i.to_s
+def get_vm_name(i, v)
+  File.basename(Dir.getwd) + "-ml-v" + v + "-" + i.to_s
```

Comment:
Can you remove this from the PR, I'd like to keep version out of VM names. Or you have to make an option out of it that defaults to not including version. Doing so will make it more convenient to do a version upgrade in future..